web/src/components/data-test.ts

```diff
@@ -0,0 +1,193 @@
+export const AlertsPage = {
+  DownloadCSVButton: 'download-csv-button',
+  AlertingRuleRow: {
+    AlertingRuleArrow: 'alerting-rule-arrow',
+    AlertingRuleResourceIcon: 'alerting-rule-resource-icon',
+    AlertingRuleResourceLink: 'alerting-rule-resource-link',
+    AlertingRuleSeverityBadge: 'alerting-rule-severity-badge',
+    AlertingRuleTotalAlertsBadge: 'alerting-rule-total-alerts-badge',
+    AlertingRuleStateBadge: 'alerting-rule-state-badge',
+  },
+  AlertRow: {
+    AlertResourceIcon: 'alert-resource-icon',
+    AlertResourceLink: 'alert-resource-link',
+    AlertSeverityBadge: 'alert-severity-badge',
+    AlertNamespace: 'alert-namespace',
+    AlertState: 'alert-state',
+    AlertSource: 'alert-source',
+    AlertCluster: 'alert-cluster',
+    SilenceAlertDropdownItem: 'silence-alert-dropdown-item',
+  },
+};
+
+export const AlertingRulesPage = {
+  AlertingRuleRow: {
+    AlertingRuleResourceIcon: 'alerting-rule-resource-icon',
+    AlertingRuleResourceLink: 'alerting-rule-resource-link',
+    AlertingRuleSeverityBadge: 'alerting-rule-severity-badge',
+    AlertingRuleStateBadge: 'alerting-rule-state-badge',
+    AlertingRuleSource: 'alerting-rule-source',
+  },
+};
+
+export const AlertRulesDetailsPage = {
+  Header: {
+    AlertResourceIcon: 'alert-resource-icon',
+    SeverityBadgeHeader: 'severity-badge-header',
+    AlertingRulesBreadcrumb: 'alerting-rules-breadcrumb',
+  },
+  Details: {
+    Name: 'name',
+    Severity: 'severity-badge-details',
+    Description: 'description',
+    Summary: 'summary',
+    Source: 'source',
+    For: 'for',
+    Expression: 'expression',
+    Labels: 'labels',
+  },
+  ActiveAlerts: {
+    Description: 'active-alerts',
+  },
+};
+
+export const AlertingDetailsPage = {
+  Header: {
+    AlertResourceIcon: 'alert-resource-icon',
+    SeverityBadgeHeader: 'severity-badge-header',
+    AlertingBreadcrumb: 'alerting-breadcrumb',
+    SilenceAlertButton: 'silence-alert-button',
+  },
+  Details: {
+    Name: 'name',
+    Severity: 'severity',
+    Description: 'description',
+    Summary: 'summary',
+    Source: 'source',
+    State: 'state',
+    Labels: 'labels',
+    AlertingRuleIcon: 'alerting-rule-icon',
+    AlertingRuleLink: 'alerting-rule-link',
+  },
+};
+
+export const SilencesPage = {
+  CreateSilenceButton: 'create-silence-btn',
+  ExpireXSilencesButton: 'expire-x-silences-button',
+  SilencesTable: {
+    SilencesTable: 'silences-table',
+    SelectAllCheckbox: 'select-all-checkbox',
+    SelectCheckbox: 'select-checkbox',
+    SilenceResourceIcon: 'silence-resource-icon',
+    SilenceResourceLink: 'silence-resource-link',
+    SilenceMatchersList: 'silence-matchers-list',
+    SilenceCount: 'silence-count',
+    SilenceState: 'silence-state',
```

Comment:
This id is duplicated in other page, I think the purpose of this file is to reuse these ids. We could use a flat structure and create a single object that holds the unique ids. Otherwise it will become difficult to know from which id is this `silence-state`